framework/OpenMod.API/Console/IConsoleActor.cs

```diff
@@ -0,0 +1,9 @@
+﻿using OpenMod.API.Commands;
+
+namespace OpenMod.API.Console
+{
+    public interface IConsoleActor : ICommandActor
```

Comment:
There is no need for this interface, since we have ICommandActor.Type which can be used to check if it's a console. There is also no need to check if its a game console.